ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedNodesManager.java

```diff
@@ -334,7 +335,8 @@ public void onEventDispatch(Event event) {
       EventAnimationDriver eventDriver = mEventDrivers.get(event.getViewTag() + eventName);
       if (eventDriver != null) {
         event.dispatch(eventDriver);
-        mUpdatedNodes.put(eventDriver.mValueNode.mTag, eventDriver.mValueNode);
+
+        updateNodes(Collections.singletonList((AnimatedNode) eventDriver.mValueNode));
```

Comment:
There are cases where this won't happen in the same frame, like during momentum scrolling, this is why we need to update it immediatly instead of relying on runUpdates which is called on each frame by the Choreographer.

These functions only update nodes that are marked as needing an update so it won't update twice in a frame.